rails/blueprints/api_versioner/USAGE

```diff
@@ -0,0 +1,14 @@
+Description:
```

Comment:
Love the idea of a usage file! We should probably go back and add these to all templates.